en/docs/api-analytics/samples/publishing-custom-analytics-data.md

```diff
@@ -0,0 +1,193 @@
+
+    <repository>
+        <id>wso2-nexus</id>
+        <name>WSO2 internal Repository</name>
+        <url>https://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+        <releases>
+            <enabled>true</enabled>
+            <updatePolicy>daily</updatePolicy>
+            <checksumPolicy>ignore</checksumPolicy>
+        </releases>
+    </repository>
+
+Add dependencies,
+
+    <dependency>
+        <groupId>org.wso2.carbon.apimgt</groupId>
+        <artifactId>org.wso2.carbon.apimgt.gateway</artifactId>
+        <version>${carbon.apimgt.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.wso2.carbon.apimgt</groupId>
+        <artifactId>org.wso2.carbon.apimgt.common.analytics</artifactId>
+        <version>${carbon.apimgt.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.synapse</groupId>
+        <artifactId>synapse-extensions</artifactId>
+        <version>${synapse.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.synapse</groupId>
+        <artifactId>synapse-core</artifactId>
+        <version>${synapse.version}</version>
+        <exclusions>
+            <exclusion>
+                <groupId>org.apache.axis2</groupId>
+                <artifactId>axis2-codegen</artifactId>
+            </exclusion>
+            <exclusion>
+                <groupId>org.wso2.orbit.com.github.fge</groupId>
+                <artifactId>json-schema-validator-all</artifactId>
+            </exclusion>
+        </exclusions>
+    </dependency>
+
+!!! Info
+
+	- The versions for ${carbon.apimgt.version}, and ${synapse.version} can be found in the jar versions available in the current API Manager 4.0.0 package.
+    - Since this feature support available in the latest update levels, you have to follow the below steps to add the required artifacts to the local m2 repository and point it as a repository in pom file.
+        1. Find the `org.wso2.carbon.apimgt.gateway` and `org.wso2.carbon.apimgt.common.analytics` jars inside the `<APIM_HOME>/repository/components/plugins` directory in latest U2 updated APIM pack.
+        2. Add the above jars to the local m2 manually.
+        ```code
+        mvn install:install-file -Dfile=<PATH_TO_FILE>/org.wso2.carbon.apimgt.gateway_<COMPENENT_VERSION>.jar -DgroupId=org.wso2.carbon.apimgt -DartifactId=org.wso2.carbon.apimgt.gateway -Dversion=<COMPENENT_VERSION> -Dpackaging=jar
+        mvn install:install-file -Dfile=<PATH_TO_FILE>/org.wso2.carbon.apimgt.common.analytics_<COMPENENT_VERSION>.jar -DgroupId=org.wso2.carbon.apimgt -DartifactId=org.wso2.carbon.apimgt.common.analytics -Dversion=<COMPENENT_VERSION> -Dpackaging=jar
+        ```
+        3. Point local m2 repository in project pom
+        ```code
+        <repository>
+            <id>local-maven-repo</id>
+            <url>file://home/user/.m2/repository</url>
+        </repository>
```

Comment:
The formatting looks a bit messy here. Shall we properly align this section?

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/6104/commits/1a243d5c12d13171c6a00eecdba75d374c2bc77c
Thanks